mainnet1/mainnet1.toml

```diff
@@ -63,7 +63,7 @@ Value = "mainnet"
       # "https://vega.greenfield.xyz/graphql",
       # !! not passing standard validations
       # "https://vega.xprv.io/datanode/query",
-      # "https://vega.mainnet.stakingcabin.com/query",
+      "https://vega.mainnet.stakingcabin.com/query",
```

Comment:
This one doesn't work for me, but `https://vega.mainnet.stakingcabin.com:3008/query` works.

Reply:
changed, thanks